pulpcore/cli/common/generic.py

```diff
@@ -82,18 +101,21 @@ def __init__(self, *args: Any, **kwargs: Any) -> None:
         ).strip()
         super().__init__(*args, **kwargs)
 
-    def handle_parse_result(self, ctx: click.Context, opts: Dict[str, Any], args: Any) -> Any:
+    def handle_parse_result(
+        self, ctx: click.Context, opts: Mapping[str, Any], args: List[Any]
```

Comment:
Why did this go from Dict to Mapping?

Reply:
Because the function we inherit it from is using Mapping (which is more general).